File: TFC-TourGuide-PHP/modelos/Turista.php
<?php

// Esta clase representa una Usuario

class Turista
{
    // Variables de clase
    private $id,$token ,$nombre, $apellidos,$ciudad,$email,$puntuacion;

    // Constructor
    public function __construct($token,$nombre, $apellidos,$ciudad,$email,$puntuacion)
    {
        $this->token = $token;
        $this->nombre = $nombre;
        $this->apellidos = $apellidos;
        $this->ciudad = $ciudad;
        $this->email = $email;
        $this->puntuacion = $puntuacion;
    }

    public function getToken(){
        return $this->token;
    }

    public function getNombre(){
        return $this->nombre;
    }

    public function getApellidos(){
        return $this->apellidos;
    }
    
    public function getCiudad(){
        return $this->ciudad;
    }

    public function getEmail(){
        return $this->email;
    }

    public function getPuntuacion(){
        return $this->puntuacion;
    }

    // Muestra los datos de la Usuario
    public function toString()
    {
        return
            [
                "id" => utf8_encode($this->id),
                "nombre" => utf8_encode($this->nombre),
                "apellido1" => utf8_encode($this->getApellidos),
                "email" => utf8_encode($this->email),
                "puntuacion" => utf8_encode($this->puntuacion),
                "token" => utf8_encode($this->token),
            ];
    }
}

File: TFC-TourGuide-Android/app/src/main/java/com/example/tourguide/modelo/Turista.java
package com.example.tourguide.modelo;

public class Turista {
    private String id,token,nombre, apellidos, email;
    private Double puntuacion;

    public Turista(String id,String token, String nombre, String apellidos,String email,Double puntuacion) {
        this.id = id;
        this.token = token;
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.email = email;
        this.puntuacion = puntuacion;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellido1(String apellido1) {
        this.apellidos = apellidos;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public Double getPuntuacion() {
        return puntuacion;
    }

    public void setPuntuacion(Double puntuacion) {
        this.puntuacion = puntuacion;
    }

    @Override
    public String toString() {
        return "Turista{" +
                "id=" + id + '\''+
                "nombre='" + nombre + '\'' +
                ", apellidos='" + apellidos + '\'' +
                ", email='" + email + '\'' +
                ", puntuacion=" + puntuacion +
                '}';
    }



}
